microsvcs/user/app/controller/profile_con.js

```diff
@@ -7,33 +8,30 @@ const QUERIES = require("../model/profile.js")
  * 1 request parameter expected.
  * @param {String} uid - unique uid field
  */
-exports.getProfileByUID = async (req, res) => {
+exports.getProfileByUID = async (req, res, next) => {
```

Comment:
I believe next is only for error handling middleware, please verify on that

Reply:
okay seems like you have two middleware, maybe thats why. But try to verify on that